documentation/docs/libraries/rust/how_to/native_tokens/_04_burn_native_token/_expected_output.md

```diff
@@ -0,0 +1,71 @@
+```plaintext {10-14,45-49}
```

Comment:
Curious, what's that ?

Reply:
it highlights lines 10 to 14 and 45 to 49

![image](https://user-images.githubusercontent.com/85233773/191734788-f389ef12-ae98-486b-a76c-5fcc46ec543a.png)

Reply:
Thanks